app/controllers/further_education_payments/providers/claims/verifications_controller.rb

```diff
@@ -3,7 +3,15 @@ module Providers
     module Claims
       class VerificationsController < BaseController
         before_action :authorise_claim!
-        before_action :set_form
+        before_action :set_form, except: %i[show]
+        before_action :redirect_if_verified, except: %i[show]
```

Comment:
not a blocker but i think filters are easier to comprehend when inverted to `only` as opposed to `except`